autoria.py

```diff
@@ -399,11 +399,26 @@ def select_item(item_to_select: str, items_list: dict) -> int:
             needed item (category, bodystyle, mark etc.) identifyer.
         """
         if item_to_select is not None and items_list is not None:
+            result = None
             for item in items_list:
-                if fnmatch(item['name'], '*' + item_to_select + '*'):
-                    return item['value']
-        else:
-            return None
+                if fnmatch(item['name'], item_to_select):
+                    result = item['value']
+                    print(result)
+                    return result
+            if result is None:
+                for item in items_list:
```

Comment:
I thought about this a lot... It seems that I do need to check through whole list again and again... On the same example of Scenic, it seems that the list of dictionaries for models is sorted by id (`['value']`), so if I leave only one `for` loop, it acts the same way if I had only one check with wild-carded condition, it picks "Grand Scenic" instead of just "Scenic" (I'm searching for "Sceni"), and works good if I have three loops.